src/index.ts

```diff
@@ -4,103 +4,31 @@
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-import path from "path";
-import fs from "fs";
-import { Command, flags } from "@oclif/command";
-import { validateFile, printResults } from "./validator";
 
-export default class RamlToolkitCommand extends Command {
-  async run(): Promise<void> {
-    const { argv, flags } = this.parse(RamlToolkitCommand);
+// This preserves the linter as the default oclif command
+import lint from "./lint";
+export default lint;
```

Comment:
```suggestion
export { lint as default } from "./lint";
```

Reply:
Yea, it doesn't work because we're importing the default, not a named import (and oclif expects things a certain way so it needs to be default)